client/deposits/strings.ts

```diff
@@ -32,3 +32,106 @@ export const depositStatusLabels: Record<
 	canceled: __( 'Canceled', 'woocommerce-payments' ),
 	failed: __( 'Failed', 'woocommerce-payments' ),
 };
+
+/**
+ * Mapping of payout failure code to display string.
+ */
+export const payoutFailureMessages: Record< PayoutFailureCode, string > = {
+	insufficient_funds: __(
+		'Your account has insufficient funds to cover the transfer.',
+		'woocommerce-payments'
+	),
+	bank_account_restricted: __(
+		'The bank account has restrictions on either the type or number of transfers allowed. This normally indicates that the bank account is a savings or other non-checking account.',
+		'woocommerce-payments'
+	),
+	debit_not_authorized: __(
+		'Debit transactions are not approved on the destination bank account. Bank accounts need to be set up for both credit and debit transfers.',
+		'woocommerce-payments'
+	),
+	invalid_card: __(
+		'The card used was invalid. This usually means the card number is invalid or the account has been closed. Please verify card details before retrying.',
+		'woocommerce-payments'
+	),
+	declined: __(
+		'The bank has declined this transfer. Please contact the bank before retrying.',
+		'woocommerce-payments'
+	),
+	invalid_transaction: __(
+		'The transfer was refused by the issuing bank because this type of payment is not permitted for this card. Please contact the issuing bank for clarification.',
+		'woocommerce-payments'
+	),
+	refer_to_card_issuer: __(
+		'The transfer was refused by the card issuer. Please contact the issuing bank for clarification before trying again.',
+		'woocommerce-payments'
+	),
+	unsupported_card: __(
+		'The bank no longer supports transfers to this card.',
+		'woocommerce-payments'
+	),
+	lost_or_stolen_card: __(
+		'The card used has been reported lost or stolen. Please contact the issuing bank for clarification.',
+		'woocommerce-payments'
+	),
+	invalid_issuer: __(
+		'The issuer specified by the card number does not exist. Please verify card details before retrying.',
+		'woocommerce-payments'
+	),
+	expired_card: __(
+		'The card used has expired. Please switch to a different card or payment method. Contact the issuing bank for clarification before trying again.',
+		'woocommerce-payments'
+	),
+	could_not_process: __(
+		// The same failure code is used if processing is failed by the bank or Stripe.
+		'The bank or the payment processor could not process this transfer.',
+		'woocommerce-payments'
+	),
+	invalid_account_number: __(
+		'The bank account details on file are probably incorrect. The routing number seems correct, but the account number is invalid.',
+		'woocommerce-payments'
+	),
+	incorrect_account_holder_name: __(
+		'The bank account details on file are probably incorrect.',
```

Comment:
I ran through these sentences to AI and this one can be improved:

```suggestion
		'The bank account holder name on file appears to be incorrect.',
```

Reply:
Changed here, thanks - [d256e22](https://github.com/Automattic/woocommerce-payments/pull/10462/commits/d256e22ea41da96714e462291a2032cd16427eb0)